src/features/events/components/ParticipantSummaryCard.tsx

```diff
@@ -145,44 +147,64 @@ const ParticipantSummaryCard: FC<ParticipantSummaryCardProps> = ({ model }) => {
               )}
             </Box>
           </Box>
-          <Box display="flex" flexDirection="column">
-            <Typography color={'secondary'}>
-              {messages.participantSummaryCard.booked()}
-            </Typography>
-            <Box display="flex">
-              <Typography variant="h4">{`${remindedParticipants}/${availParticipants}`}</Typography>
-              {remindedParticipants < availParticipants && (
-                <Tooltip
-                  arrow
-                  placement="top-start"
-                  title={
-                    contactPerson == null
-                      ? messages.participantSummaryCard.remindButtondisabledTooltip()
-                      : ''
-                  }
+          {new Date(eventData.start_time) > new Date() ? (
```

Comment:
I find it very difficult to read these very large ternary expressions in the middle of JSX, and would appreciate if you can shorten it in one of several ways:

### 1. Separate components
Convert the contents to separate components (defined in the same module) and use as such:

```tsx
{new Date(eventData.start_time) > new Date() ? <ReminderInfo …/> : <AttendanceInfo …/>}
```

### 2. Functions
Move the contents to functions (defined inside of this component) and use as such:

```tsx
{new Date(eventData.start_time) > new Date() ? renderReminderInfo() : renderAttendanceInfo()}
```

Reply:
I started working on this but decided to leave it as I think this component could do with a bigger refactor than what I have time to do right now.